docs/en/plugins/server.md

```diff
@@ -61,4 +61,59 @@ server:
   #
   # Default: 60s
   relay_timeout: 60s
-```
\ No newline at end of file
+```
+
+### Server initialization
+
+The `on_init` section is used for application initialization or warming up before starting workers. It allows you to set
+a command script that will be executed before starting the workers. You can also set environment variables to pass to
+this script.
+
+### Worker starting command
+
+The `server.command` option is required and is used to start the worker pool for each configured section in the config.
+
+> **Note:**
+> This option can be overridden by plugins with a pool section, such as the http.pool.command.
+
+The `user` and `group` options allow you to set the user and group that will start and own the worker process. An empty
+value means to use the RoadRunner process user.
+
+The `env` option allows you to set environment variables to pass to the worker script.
+
+## PHP Client
+
+There is a package that simplifies the process of integrating the RoadRunner worker pool with a PHP application. The
+package contains a common codebase for all RoadRunner workers, making it easy to integrate and communicate with workers
+created by RoadRunner.
+
+When RoadRunner creates workers for any plugin that uses workers, it runs a PHP script and starts communicating with it
+using a relay. The relay can be `pipes`, `TCP`, or a `socket`. The PHP client simplifies this process by providing a
```

Comment:
Might also mention, that `pipes` is the fastest communication transport and used by default.